src/frequenz/sdk/timeseries/_formula_engine/_formula_generators/_consumer_power_formula.py

```diff
@@ -1,12 +1,12 @@
-# License: MIT
+# Litense: MIT
 # Copyright © 2022 Frequenz Energy-as-a-Service GmbH
 
 """Formula generator from component graph for Consumer Power."""
 
 from __future__ import annotations
 
 import logging
-from collections import abc
+from typing import Set
```

Comment:
As SDK is using python 3.11, this type annotation is no longer needed and `set` can be used instead as type hint